Mathlib/FieldTheory/Finite/Basic.lean

```diff
@@ -533,6 +533,11 @@ theorem pow_card_sub_one_eq_one {p : ℕ} [Fact p.Prime] {a : ZMod p} (ha : a 
     rwa [ZMod.card p] at h
 #align zmod.pow_card_sub_one_eq_one ZMod.pow_card_sub_one_eq_one
 
+lemma pow_card_sub_one (a : ZMod p) : a ^ (p - 1) = if a ≠ 0 then 1 else 0 := by
```

Comment:
```suggestion
lemma pow_card_sub_one {p : ℕ} [Fact p.Prime] (a : ZMod p) : a ^ (p - 1) = if a ≠ 0 then 1 else 0 := by
```

Reply:
Whoops indeed. I am editing mathlib cache-less since [this](https://leanprover.zulipchat.com/#narrow/stream/287929-mathlib4/topic/cache.20misses.20aesop) happened